pkg/config/config.go

```diff
@@ -51,6 +51,7 @@ const (
 	DefaultAggregationGranularity = "1h"
 	DefaultLayerRetentionTime     = "1y"
 	DefaultSampleRetentionTime    = 0
+	DefaultLogLevel               = "info"
```

Comment:
if you are adding this variable please use it in the tsdbctl.go (line 195) for the default log level, otherwise there is no point in it

Reply:
I already use `DefaultLogLevel` in **tsbdct.go** — on line 64 see (https://github.com/v3io/v3io-tsdb/pull/172/files#diff-e82ef7d65c5957422d23f4a3ea81f257R64 — just scroll to the end):
```
"Verbose output. Add \"=<level>\" to set the log level -\ndebug | info | warn | error. For example: -v=warn.\n(default - \""+config.DefaultVerboseLevel+"\" when using the flag; \""+config.DefaultLogLevel+"\" otherwise)")
```

Reply:
See new commit eafd800 — use the `config.DefaultLogLevel` flag to set the default log level in the code (**tsdbctl.go** line 195):
`	cfg.LogLevel = config.DefaultLogLevel`